manifests/o/Oracle/VirtualBox/6.1.40/Oracle.VirtualBox.installer.yaml

```
@@ -0,0 +1,40 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Oracle.VirtualBox
+PackageVersion: 6.1.40
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+  InstallLocation: --msiparams "INSTALLDIR="<INSTALLPATH>""
+  Custom: -msiparams REBOOT=ReallySuppress
+InstallerSuccessCodes:
+- 3010
+UpgradeBehavior: install
+FileExtensions:
+- ova
+- ovf
+- vdi
+- vhd
+- vmdk
+AppsAndFeaturesEntries:
+- DisplayName: Oracle VM VirtualBox 6.1.38

```

```suggestion
- DisplayName: Oracle VM VirtualBox 6.1.40
```

This needs to match Control Panel